frontend/src/views/ResultScreenView.vue

```diff
@@ -0,0 +1,61 @@
+<script setup lang="ts">
+import { ref, onMounted } from 'vue'
+import { useLevelStore } from '@/stores/levels'
+import type { Level } from '@/domain/Level'
+import { ErrorMessage } from '@/services/messages'
+
+const currentLevel = ref<Level | null>(null)
+const nextLevel = ref<Level | null>(null)
+const levelStore = useLevelStore()
+
+onMounted(async () => {
+  const roleId = 1
+
+  try {
+    const scoreResponse = await levelStore.getScoreResponse(roleId)
+
+    if (scoreResponse) {
+      currentLevel.value = scoreResponse.currentLevel
```

Comment:
Instead of putting this in a ref, set in the state. Then you can use the same information globally across the application